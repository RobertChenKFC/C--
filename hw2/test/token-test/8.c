../assignment-test/8.c