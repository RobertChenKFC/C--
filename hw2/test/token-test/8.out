IDENTIFIER
IDENTIFIER
IDENTIFIER
COMMENT
/****  start ***/
[Error] Lexer: invalid identifier this_identifier_is_too_loooooooooooooooooooooooooooooooooooooooooooooong of length > 32 on line 7
